packages/core/src/new-api/internal/new-execution/jsonrpc-calls.ts

```diff
@@ -0,0 +1,600 @@
+import { IgnitionError } from "../../../errors";
```

Comment:
This includes all the RPC calls that we were already doing. This replaces both the adapters and chain dispatcher.